src/components/common/ui/CourseCard/CourseCard.tsx

```diff
@@ -129,19 +124,15 @@ const CourseCard: React.FC<CourseCardProps> = ({
         )}
       </div>
       <div className={styles.tagRating}>
-        <Link href={`/preview-courses/${id}`} className={styles.button}>
-          <Button
-            variant="contained"
-            color="error"
-            style={{
-              width: 'calc(100% - 80px)',
-              marginTop: '20px',
-            }}
-            endIcon={<ArrowCircleRightIcon />}
-          >
-            Explore
-          </Button>
-        </Link>
+        <Button
```

Comment:
For accessibility rules, all links that redirect to another page (even if it's SPA) should be links, but not buttons. I found [this video](https://youtu.be/ssJsjGZE2sc?t=957) special for you with the right timecode :) I highly recommend watching the whole video!